tests/auto/mocks/declarativewebutils/declarativewebutils.cpp

```diff
@@ -55,21 +55,11 @@ void DeclarativeWebUtils::updateWebEngineSettings()
 
     // Add only mandatory preferences for unit tests.
 
-    // Don't force 16bit color depth
-    webEngineSettings->setPreference(QString("gfx.qt.rgb16.force"), QVariant(false));
-
     // Use external Qt window for rendering content
     webEngineSettings->setPreference(QString("gfx.compositor.external-window"), QVariant(true));
     webEngineSettings->setPreference(QString("gfx.compositor.clear-context"), QVariant(false));
     webEngineSettings->setPreference(QString("embedlite.compositor.external_gl_context"), QVariant(true));
     webEngineSettings->setPreference(QString("embedlite.compositor.request_external_gl_context_early"), QVariant(true));
-
```

Comment:
For reference: https://git.sailfishos.org/mer-core/gecko-dev/blob/sailfishos-esr52/embedding/embedlite/embedding.js#L194